utils/bash/ci.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
That's fine.

What we can do, then, is write our own shell script invocation method that sources that file before invoking the shell script. Something like:

```ruby
def dd_sh(script)
  sh('source ci.sh && ${script}')
end
```